loops.js

```diff
@@ -18,6 +20,13 @@
 // klaar!
  // ==========================================
 
+for(let i = 0; i < 4; i++) {
```

Comment:
Volgens de uitwerkingen van Nova is dit het juiste code:

for (let i = 0; i < 5; i++) {
  if (i === 4) {
    console.log('klaar!')
  } else {
    console.log('loop...');
  }
}